<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/iLogos/nsst.jpg" width="400" height="100">

# DEMI | NOVA SST

## Mecânica dos Sólidos I

## Tração e compressão de barras elásticas
## Vigas sujeitas à tracção/compressão. Parte I
## Problema 4 <a name="PIV84"></a>

O parafuso de alumínio de 10 mm de diâmetro representado na figura é colocado dentro de um tubo de aço que tem diâmetro externo de 18 mm e paredes de 3 mm de espessura. Inicialmente não há quaisquer tensões no parafuso e no tubo, estando a porca perfeitamente ajustada de forma a não ocorrer folga entre ambos. Em seguida dá-se à porca um aperto de um quarto de volta e o conjunto é submetido a um aumento de temperatura de 100 ºC. Sabendo que o passo da rosca é de 2 mm, $E_\mathrm{aço} = 200$ GPa, $E_\mathrm{alumínio}$ = 70 GPa, $\alpha_\mathrm{aço}$ = 18 $\times 10^{-6}$ /ºC, $\alpha_\mathrm{alumínio}$ = 23 $\times 10^{-6}$ /ºC, determine as tensões normais que vão surgir no parafuso e no tubo.

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P004_i1.jpg"
width="500" height="260">



## Resolução

A resistência e a rigidez são considerados frequentemente para efeitos de dimensionamento de componentes mecânicos ou estruturas em engenharia. O nível de tensão aplicado é frequentemente usado como medida de resistência. Contudo, o dimensionamento pode contemplar também um critério em deslocamento.

A rigidez está relacionada com a capacidade de uma peça resistir ao deslocamento (deflexão) ou deformação. Esta grandeza é usada frequentemente como fator de controlo em projeto mecânico ou estrutural. O deslocamento relativo ou folgas entre partes, por exemplo, pode ser prescrito dentro de certos limites de projeto. Elementos de máquinas ou estruturas devem ter uma rigidez adequados para manter a precisão de fabrico em serviço. Ou componentes de máquinas podem exigir grande rigidez para eliminar problemas de vibração.

### Deslocamento de membros com deformação axial de origem mecânica

Considera-se um elemento retilíneo de secção transversal constante $A$, comprimento $L$, constituido de um material com módulo de Young $E$, e submetido a carregamento axial, $P$. O valor da tensão nominal axial é $\sigma_{xx} =
P/A = E \varepsilon_{xx} $ e o valor da extensão linear  $\varepsilon_{xx} = \delta/L$, pelo que, a variação de comprimento do elemento é dada por:

\begin{equation*}
\delta_P = \frac{PL}{AE}\quad\textrm{(mm)}
\end{equation*}

o produto $AE$ é designado por rigidez axial da barra ou elemento; o subscrito $P$ em $\delta_P$ indica que o deslocamento axial tem como origem uma força aplicada. O sinal positivo para $\delta_P$ indica alongamento (tração: $+\delta_P$), enquanto que um sinal negativo representaria contração (compressão: $-\delta_P$).

### Deslocamento de membros com deformação axial de origem térmica

Uma variação de temperatura $\Delta T$ num elemento irá causar uma deformação do mesmo dada por: $\varepsilon_{t} = \alpha \Delta T$, em que $\alpha$ representa o coeficiente de expansão térmico (unidade: por grau
Celcius, ºC $^{-1}$). Num corpo elástico, a variação de comprimento por variação térmica é dado por:

\begin{equation*}
\delta_t = \alpha (\Delta T) L \quad\textrm{(mm)}
\end{equation*}

A deformação de origem térmica é geralmente positiva se a temperatura aumentar e negativo se esta diminuir.

### Análise do deslocamento relativo no aperto parafuso-porca

Considere-se o sistema de fixação parafuso/porca. A rosca tem um passo (distância entre pontos homólogos de filetes consecutivos) de $p=2$ mm. De acordo com o enunciado, a porca tem um avanço (distância axial percorrida pela rosca) de um quarto (1/4) de volta (distância axial numa volta completa que igualará o passo da rosca):

\begin{equation*}
\Delta  = \frac{1}{4}p = \frac{1}{4}(2) = 0.5~\textrm{mm}
\end{equation*}

O avanço da porca gera uma força de compressão no tubo de aço responsável por um deslocamento axial do tubo, $\delta_t$. Se o parafuso fosse rígido e indeformável ($E_\mathrm{parafuso} = \infty$):

\begin{equation*}
\Delta  \equiv \delta_t
\end{equation*}

contudo na prática, o parafuso tem uma rigidez finita e, em consequência, deforma-se pela ação de uma força que o tubo exerce sobre a ligação parafuso/porca ou consequentemente sobre este, pelo que:

\begin{equation*}
|\Delta|  \gt |\delta_t|
\end{equation*}

A equação de compatibilidade de deslocamentos, entre as configurações inicial e a final, é expressa por:

\begin{equation*}
|\Delta - \delta_p^{(F_p)}| = |\delta_t^{(F_t)}|
\end{equation*}

ou,

\begin{equation*}
- \Delta + \delta_p^{(F_p)} = -\delta_t^{(F_t)}~(\textrm{em que, o sinal negativo indica compressão e o sinal positivo tração})
\end{equation*}

<img src="https://userweb.fct.unl.pt/~jmc.xavier/MSI/Notebooks/Au08/Au08_P004_i3.jpg"
width="650">

Note-se que o aperto introduzido pelo avanço da porta, no sistema de fixação parafuso/porca, irá exercer uma força de compressão no tubo. O tubo, por sua vez, irá gerar uma força de tração no sistema de fixação.

Seja $F_t$ a força exercida sobre o tubo pelo sistema de fixação roscado (parafuso/porta) e $F_p$ a força que é exercida sobre o parafuso pelo tubo em compressão. As equações de equilíbrio permitem concluir que:

\begin{equation*}
F_p = F_t
\end{equation*}

pelo que o problema é estaticamente indeterminado de primeiro grau. Desenvolvendo a equação de compatibilidade dos deslocamentos vem:

\begin{equation*}
\Delta - \delta_p^{(F_p)} = \delta_t^{(F_t)}
\quad\Leftrightarrow\quad
\Delta = \delta_t^{(F_t)} + \delta_p^{(F_p)}
\quad\Leftrightarrow\quad
\Delta = \frac{F_tL_t}{A_tE_t} + \frac{F_pL_p}{A_pE_p}
\end{equation*}

cujo desenvolvimento algébrico conduz a ($F = F_t = F_p$ e $L = L_t = L_p$):

\begin{equation*}
\Delta = FL\left(\frac{1}{A_tE_t} + \frac{1}{A_pE_p}\right)
\quad\Leftrightarrow\quad
F = \left(\frac{A_tA_pE_tE_p}{A_pE_p + A_tE_t} \right)\frac{\Delta}{L}
\end{equation*}

com,

\begin{equation*}
A_t = \phi^\mathrm{ext} - \phi^\mathrm{int} = \frac{\pi}{4}(d^2_\mathrm{ext}-d^2_\mathrm{int}) 
\quad\wedge\quad
A_p = \frac{\pi d}{4}
\end{equation*}

In [2]:
import numpy as np

# units: mm, N, MPa

Ep, Et = 70.e3, 200.e3 # unidade: MPa

Lp, Lt = 400., 400. # unidade: mm
L = Lp
dp, dtext, dtint, passo = 10., 18., 12., 2.
Delta = passo/4

At = np.pi/4*(dtext**2 - dtint**2)
Ap = np.pi/4*(dp**2)
F = Delta/L*(At*Ap*Et*Ep)/(Ap*Ep + At*Et)
sigt = -F/At
sigp = F/Ap

print('- tubo:')
print(f'At   = {At:.2f} mm^2')
print(f'Ft   = {-F:.2f} N (C, compressão)')
print(f'sigt = {sigt:.2f} MPa (C, compressão)')

print('\n- parafuso:')
print(f'Ap   = {Ap:.2f} mm^2')
print(f'Fp   = {F:.2f} N (T, tração)')
print(f'sigp = {sigp:.2f} MPa (T, tração)')

- tubo:
At   = 141.37 mm^2
Ft   = -5753.50 N (C, compressão)
sigt = -40.70 MPa (C, compressão)

- parafuso:
Ap   = 78.54 mm^2
Fp   = 5753.50 N (T, tração)
sigp = 73.26 MPa (T, tração)


O efeito do avanço da porca do sistema de fixação provoca tensões de tração no parafuso e tensões de compressão no tubo:

\begin{equation*}
\begin{aligned}
\sigma_t &= \frac{F}{A_t} = 40.70~\textrm{(MPa) C, Compressão} \\
\sigma_p &= \frac{F}{A_p} = 73.25~\textrm{(MPa) T, tração} \\
\end{aligned}
\end{equation*}

#### Efeito combinado do avanço da porca do sistema de fixação + variação da temperatura

Ocorrendo uma variação de temperatura $\Delta T$ existirá uma variação de comprimento de origem térmica quer no parafuso quer no tubo ($L_p = L_t = L$):

\begin{equation*}
{\delta_p}^T = \alpha_p (\Delta T) L \quad\wedge\quad
{\delta_t}^T = \alpha_t (\Delta T) L
\end{equation*}

Pela ação simultânea da força de ligação parafuso/porca e da variação de temperatura surgirá pelo princípio de sobreposição a seguinte equação de compatibilidade de deslocamentos do parafuso e tubo:

\begin{equation*}
\underbrace{-\Delta + \delta_p^F + \delta_p^T}_{\mathrm{parafuso}} =
\underbrace{-\delta_t^F + \delta_t^T}_{\mathrm{tubo}}
\end{equation*}

em que os sinais positivos para $\delta_p^F$, $\delta_p^T$ e $\delta_t^T$ indicam alongamento e o sinal nevativo para $\Delta$ e $\delta_t^F$ indica contração devido ao carregamento de compressão imposto no aperto.

De notar que se o material do tubo e parafuso tiverem o mesmo coefiente de expansão térmica ($\alpha_p = \alpha_t$) a equação anterior reduz-se apenas à equação de compatibilidade devido aos esforços de aperto ($-\Delta + \delta_p^F~= -\delta_t^F$), sendo que a variação de temperatura não introduz qualquer deformação adicional (i.e. deslocamentos relativos).

Desenvolvimento algébrico a expressão anterior ($F = F_t = F_p$ e $L = L_t = L_p$):

\begin{equation*}
F = \left(\frac{A_tA_pE_tE_p}{A_pE_p + A_tE_t} \right)
\left[\frac{\Delta}{L}
+ (\alpha_t-\alpha_p) \Delta T
\right]
\end{equation*}

In [5]:
alpp, alpt, DT = 23.e-6, 18.e-6, 100.
FfT = (At*Ap*Et*Ep)/(Ap*Ep + At*Et)*(Delta/L + (alpt - alpp)*DT)
print(f'F (aperto+DT) = {FfT:.2f} N')

print('- tubo:')
sigtft = FfT/At
print(f'sig tubo (aperto + DT) = {sigtft:.2f} (MPa) C, compressão')

print(f'- parafuso:')
sigpfT = FfT/Ap
print(f'sig paraf (aperto + DT) = {sigpfT:.2f} (MPa) T, tração')

F (aperto+DT) = 3452.10 N
- tubo:
sig tubo (aperto + DT) = 24.42 (MPa) C, compressão
- parafuso:
sig paraf (aperto + DT) = 43.95 (MPa) T, tração


#### Referências

[1] Ugural, Ansel C - Mechanical Design of Machine Components, Second Edition, CRC Press (2015). Problema resolvido 4.2

---

Copyright (c) [Mecânica dos Sólidos I - DEMI - NOVA-SST]

Interactive computing by <a href="https://jupyter.org/" target="_blank"> <span
style="color:#333399"> Jupyter Notebook </span> </a> &nbsp;|&nbsp;Coded by <a href = "mailto: jmc.xavier@fct.unl.pt">José Xavier</a>

Licensed under  <a href="http://creativecommons.org/licenses/by-sa/4.0/"
target="_blank"> <span style="color:#333399;font-size: 20px"> CC BY-SA 4.0  </span></a>
